examples/ramses/run_sod_amr_reflective.py

```
@@ -200,7 +196,7 @@ def convert_to_cell_coords(dic):
 
     l_0 = np.log2(base * 2)
 
-    l = -np.log2(dX / max(dX0, dX.max())) + l_0
+    l = -np.log2(dX / dX0) + l_0
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)